# Error Func
def test_func3():
    for i in range(10, -1, -1):
        print("I am running! func2: ", 10*i)
    return 0